src/collection.js

```diff
@@ -308,15 +357,25 @@ export default class Collection {
     if (!record.id) {
       throw new Error("A record id is required.");
     }
-    const reqOptions = this._collOptions(options);
-    const { permissions } = reqOptions;
+    const { permissions } = options;
+    let updateOptions = {
+      headers: this._getHeaders(options),
+      safe: this._getSafe(options),
+    };
+    const { last_modified } = { ...record, ...options };
+    if (last_modified) {
+      updateOptions = { ...updateOptions, last_modified };
+    }
+    if (options.patch) {
+      updateOptions = { ...updateOptions, patch: options.patch };
+    }
```

Comment:
Same thing as above with `calledWith`. But I don't mind updating the tests if you think that's better.